cropper/build.gradle

```diff
@@ -9,7 +12,7 @@ android {
         minSdkVersion 14
         targetSdkVersion rootProject.targetSdkVersion
         versionCode 1
-        versionName "1.1.0"
+        versionName "1.11.0"
```

Comment:
NOTE: Jitpack will build using GitHub releases + tag. The latest version is what it shown in the badge in README. It looks like a `1.10` tag is created  (https://github.com/CanHub/Android-Image-Cropper/releases). So the `versionName` will need to be higher I believe.

Here's what I am using for reference: https://jitpack.io/docs/

Reply:
This tag is wrong. `1.10` was a mistype from `1.1.0` 

Let's change to `1.1.1` and I will delete the wrong one.

Reply:
Fixed